infra/resources/_modules/web_apps/remote-content-func.tf

```diff
@@ -0,0 +1,115 @@
+locals {
+  remote_content = {
+    app_settings = {
+      NODE_ENV = "production"
+
+      // IO COSMOSDB
+      COSMOSDB_NAME = "db"
+      COSMOSDB_URI  = var.cosmosdb_account_api.endpoint
+
+      // REMOTE CONTENT COSMOSDB
+      REMOTE_CONTENT_COSMOSDB_NAME = "remote-content-cosmos-01"
+      REMOTE_CONTENT_COSMOSDB_URI  = var.io_com_cosmos.endpoint
+
+      // BLOB STORAGE
+      MESSAGE_CONTENT_STORAGE_CONNECTION_STRING = var.message_content_storage.connection_string
+      MESSAGE_CONTAINER_NAME                    = "message-content"
+
+      // QUEUE STORAGE
+      NOTIFICATION_QUEUE_STORAGE_CONNECTION_STRING = var.com_st_queue_uri
+      NOTIFICATION_QUEUE_NAME                      = "push-notifications"
+
+      // TMP NEW NOTIFICATION QUEUE STORAGE
+      NEW_NOTIFICATION_QUEUE_STORAGE_CONNECTION_STRING = var.com_st_connectiostring
+      NEW_NOTIFICATION_QUEUE_NAME                      = "push-notifications"
+
+      IO_COM_PUSH_NOTIFICATIONS_REDIRECT_PERCENTAGE = "0"
+
+      // REDIS
+      REDIS_URL      = var.redis_cache.hostname
+      REDIS_PORT     = var.redis_cache.port
+      REDIS_PASSWORD = var.redis_cache.access_key
+
+      // BACKEND COMMUNICATION
+      BACKEND_BASE_URL = "https://io-p-app-appbackendli.azurewebsites.net"
+      BACKEND_TOKEN    = var.appbackendli_token
+
+      // INTERNAL USE PROPERTIES
+      INTERNAL_USER_ID           = var.internal_user_id
+      RC_CONFIGURATION_CACHE_TTL = "28800"
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+    }
+  }
+}
+
+
+data "azurerm_nat_gateway" "nat_gateway" {
+  name                = "io-p-itn-ng-01"
+  resource_group_name = "io-p-itn-common-rg-01"
+}
+
+module "remote_content_func" {
+  source  = "pagopa-dx/azure-function-app/azurerm"
+  version = "~> 0.0"
+
+  environment = {
+    prefix          = var.environment.prefix
+    env_short       = var.environment.env_short
+    location        = var.environment.location
+    domain          = var.environment.domain
+    app_name        = "rmt-cntnt"
```

Comment:
Can we find a better name? 😢  @lucamancosu @lucacavallaro

Reply:
I was thinking about using `rc` instead of `rmt-cntnt`, we already used `rc` in our codebase as a shorthand for `remote content`, what's your take?